tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -211,147 +211,225 @@
     "SearchColumnsUIPickerForMatter": {
       "matterName": {
         "displayInUI": true,
-        "position": 2
+        "position": 2,
```

Comment:
Added new settings to search settings to cater the needs of both project center and matter center